projects/angular/src/emphasis/_badges.clarity.scss

```
@@ -148,13 +73,13 @@
 
   @include mixins.fixForIE10AndUp {
     .badge {
-      padding: variables.$clr_baselineRem_2px variables.$clr_baselineRem_0_125 0;
+      padding: #{tokens.$cds-global-space-2} calc(#{tokens.$cds-global-space-1} + #{tokens.$cds-global-space-2}) 0;
```

You may use the mixin calculation instead. Looks more clean.

```suggestion
      padding: #{tokens.$cds-global-space-2} #{mixins.baselinePx(3)} 0;
```